external-crates/move/crates/move-compiler/tests/move_2024/naming/positional_struct_non_positional_unpack.exp

```diff
@@ -9,8 +9,11 @@ warning[W09002]: unused variable
 error[E03013]: positional call mismatch
   ┌─ tests/move_2024/naming/positional_struct_non_positional_unpack.move:7:13
   │
+2 │     public struct Foo(u64) has copy, drop;
+  │                   --- 'Foo' is declared here
+  ·
 7 │         let Foo { y: _ } = Foo(0);
-  │             ^^^^^^^^^^^^ Invalid deconstruction. Positional struct field declarations require positional deconstruction
+  │             ^^^^^^^^^^^^ Invalid struct deconstruction. Positional struct declarations require positional deconstructions
```

Comment:
Whatever we land on for constructor/construction/pack, we should try to match here